presto-main/src/test/java/io/prestosql/operator/scalar/TestStringFunctions.java

```diff
@@ -284,6 +284,11 @@ public void testStringPosition()
         testStrPosAndPosition("", null, null);
         testStrPosAndPosition(null, null, null);
 
+        assertFunction("STARTS_WITH('foo', 'foo')", BOOLEAN, true);
+        assertFunction("STARTS_WITH('foo', 'bar')", BOOLEAN, false);
+        assertFunction("STARTS_WITH('foo_bar_baz', 'foo')", BOOLEAN, true);
+        assertFunction("STARTS_WITH('foo_bar_baz', 'bar')", BOOLEAN, false);
```

Comment:
Can you add tests for:
* strings that contain non-ascii unicode characters?
* where the string is shorter than the substring
* empty string and/or empty substring

Reply:
Added several test cases